test/e2e/gorillamux/main.go

```diff
@@ -33,4 +34,7 @@ func main() {
 
 	log.Printf("Body: %s\n", string(body))
 	_ = resp.Body.Close()
+
+	// give time for auto-instrumentation to report signal
+	time.Sleep(5 * time.Second)
```

Comment:
Nope, and this should probably watch for some clearly identifying signal that the event was flushed rather than a blind sleep. But right now I just want to get this hacked together enough for a stable ish v0.1.alpha